config/i18n.ts

```diff
@@ -33,6 +33,13 @@ const countryLocaleVariants: Record<string, LocaleObjectData[]> = {
     { code: 'en-US', name: 'English (US)' },
     { code: 'en-GB', name: 'English (UK)' },
   ],
+  ca: [
+    // { code: 'ca-AD', name: 'Català (Andorra)' },
+    { code: 'ca-ES', name: 'Català (Espanya)' },
+    { code: 'ca-valencia', name: 'Català (valencià)' },
```

Comment:
can you review the names for both? shouldn't be just `Catalá` and `Valencià`

Reply:
or `Catalá (Espanya)` and `Valencià (Spanya)`: check valencià translation for Spain.

Reply:
¿También hay Catalán andorrano? :eyes:

Reply:
El otro usuario me parece que puso Valencià (Català)

Reply:
El catalan se habla en 4 estados españa, francia, italia y andorra

Reply:
Lo cambio yo mismo y haog un commit

Reply:
```suggestion
    { code: 'ca-valencia', name: 'Valencià (Català)' },
```
LoL

Reply:
El idioma es Valenciano, se añade España para evitar la colisión con otros posibles "Valencianos" (no se dará el caso), pero para el Catalán sí (creo que es lo más correcto, la Comunidad Valenciana no está en Cataluña, territorialmente, la variante es para el país no para la variante del propio idioma, tampoco sé si es así)

Reply:
Ponlo cómo en Firefox:

![imagen](https://user-images.githubusercontent.com/6311119/221367976-221f207f-14d0-40b1-854a-5c7ee86e2b97.png)

Reply:
Ok, podemos poner Valencià (Espanya), siempre estamos a tiempo de cambiarlo si hiciera falta. No hay un estandard o codigo ISO para el valenciano, por eso tanto lio

Reply:
menudo lío, mejor que nos diga algo @bnval

Reply:
vale, pon lo que hay en Firefox y luego ya podemos cambiarlo una vez lo tengamos ne main

Reply:
upps, mejor deja España entre paréntesis, me da que Firfox no está al tanto del Catalán en Italia, Andorra e Italia...
```json
 { code: 'ca-ES', name: 'Català (Espanya)' },
 { code: 'ca-valencia', name: 'Valencià (Espanya)' },
```

Reply:
En la pagina de soft valencia sugieren 'Català (valencià)', que es la que propuso @rbnval

https://www.softvalencia.org/preguntes-mes-frequents/

Reply:
```suggestion
    { code: 'ca-valencia', name: 'Català (valencià)' },
```

Reply:
vale, también me parece ok, pero incluye España en el Catalán por si acaso

Reply:
{ code: 'ca-ES', name: 'Català (Espanya)' },
{ code: 'ca-valencia', name: 'Català (valencià)' },

Si, de un principio ya le pusimos Espanya para diferenciar